SkatkovAM/lab_3/lab_3.cpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+#include <string.h>
+
+
+class sentence_with_word {
+
+
+public:
+    int sentence_size;
```

Comment:
You can move members to private section and implement access functions where needed. Also you can overload std::ostream& operator << (std::ostream&) for sentence_with_word